addons/uom/views/uom_uom_views.xml

```diff
@@ -63,14 +42,13 @@
                 <separator/>
                 <filter string="Archived" name="inactive" domain="[('active', '=', False)]"/>
                 <group string="Group By">
-                  <filter string="Category" name="group_by_category" context="{'group_by': 'category_id'}"/>
                 </group>
             </search>
         </field>
     </record>
 
     <record id="product_uom_form_action" model="ir.actions.act_window">
-        <field name="name">Units of Measure</field>
+        <field name="name">Units &amp; Packagings</field>
```

Comment:
You should update the noncontent description as well since it refers to "category" still + I guess you no longer _must_ define a conversation rate between several Units of Measure